lib/session.js

```diff
@@ -143,12 +148,34 @@ module.exports = class Session {
     return cookieSignature.sign(this.sessionId, this[secretKey])
   }
 
+  [hash] () {
+    const sess = this
+    const str = JSON.stringify(sess, function (key, val) {
```

Comment:
This is not a good hashing algorithm. How about using a stable algorithm? So two JS objects with properties in different orders will generate the same string.

Reply:
something like https://www.npmjs.com/package/fast-json-stable-stringify?

Reply:
https://www.npmjs.com/package/safe-stable-stringify < that's the one I use everywhere. Really recommended.

Reply:
cool, will add!

Reply:
done